privacy-deidentification/src/main/java/io/konig/privacy/deidentification/repo/PersonRepository.java

```diff
@@ -0,0 +1,164 @@
+package io.konig.privacy.deidentification.repo;
+
+import java.io.IOException;
+import java.security.SecureRandom;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+import org.apache.commons.lang.RandomStringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.databind.node.TextNode;
+import com.github.fge.jsonschema.core.exceptions.ProcessingException;
+
+import io.konig.privacy.deidentification.model.Identity;
+import io.konig.privacy.deidentification.model.Person;
+import io.konig.privacy.deidentification.model.PersonData;
+import io.konig.privacy.deidentification.model.PersonKeys;
+import io.konig.privacy.deidentification.model.PersonRowMapper;
+
+@Repository
+@Transactional
+public class PersonRepository {
+
+	final static SecureRandom secureRandom = new SecureRandom();
+
+	private static final String FETCH_QUERY = "SELECT DE_IDENTIFICATION.PERSON.PERSON_DATA, DE_IDENTIFICATION.PERSON.PSEUDONYM from "
```

Comment:
As a best practice, use aliases on table names and the use aliases for disambiguation.

For instance,

```
SELECT a.person_data, b.pseudonym 
FROM de_identification.person AS a
INNER JOIN de_identification.person_email AS b
ON 
   a.pseudonym=b.person_pseudonym 
   AND trim(b.email_id)=?
```